packages/react-components/react-color-picker-preview/library/src/index.ts

```diff
@@ -0,0 +1 @@
+export * from './ColorPicker';
```

Comment:
nit: we should not add implementation as part of new project scaffold PR

Reply:
This was, also, generated. So we shouldn't add default root components, right?

Reply:
reverted